src/features/script-load.js

```diff
@@ -31,7 +31,7 @@ systemJSPrototype.createScript = function (url) {
 };
 
 // Auto imports -> script tags can be inlined directly for load phase
-var lastAutoImportUrl, lastAutoImportDeps;
+var lastAutoImportUrl, lastAutoImportDeps, lastAutoImportAutoImport;
```

Comment:
The name `lastAutoImportAutoImport` is a bit awkward.

```suggestion
var lastAutoImportUrl, lastAutoImportDeps, lastAutoImportTimeout;
```